src/main/java/org/openhab/tools/analysis/tools/CheckstyleChecker.java

```diff
@@ -113,7 +113,7 @@ public void execute() throws MojoExecutionException {
         }
 
         // Maven may load an older version, if no version is specified
-        Dependency checkstyle = dependency("com.puppycrawl.tools", "checkstyle", "7.7");
+        Dependency checkstyle = dependency("com.puppycrawl.tools", "checkstyle", "8.1");
```

Comment:
It would be great to have a way to get this version directly from the pom.xml, but as far as I know there is no other way to do it instead of reading the xml file directly...